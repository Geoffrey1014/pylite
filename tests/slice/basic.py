string = "0123456789"
print string[1:8]

list = [0, 1, 2, 3]
print list[1:3]

tuple = ('00', '11', '22', '33')
print tuple[1:3]
