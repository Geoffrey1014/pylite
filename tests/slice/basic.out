1234567
[1, 2]
(11, 22)
